moor/lib/src/runtime/query_builder/statements/insert.dart

```diff
@@ -317,8 +326,10 @@ class DoUpdate<T extends Table, D> extends UpsertClause<T, D> {
   /// been inserted, use [DoUpdate.withExcluded].
   ///
```

Comment:
Let's document this new feature, maybe something like "The optional `where` clause can be used to disable the update based on the old value. If a `where` clause is set and it evaluates to `false`, a conflict will keep the old row without applying the update".